extendeddatacrossword.go

```diff
@@ -164,6 +164,10 @@ func (eds *ExtendedDataSquare) solveCrosswordRow(
 		if noMissingData(col, r) { // not completed
```

Comment:
I think this `if` never gets through, because, despite a new share of this col being rebuilt, yet, that share hasn't been placed in the col, so, the `noMissingData` returns `false`, i.e., it is incomplete. I think we need to update the column before checking if it is complete or not i.e., https://github.com/celestiaorg/rsmt2d/blob/9ed797dd43ffbee9e5212d769f0330419093a123/extendeddatacrossword.go#L165 (it points to the branch I am working on for  https://github.com/celestiaorg/rsmt2d/issues/191). WDYT?

Reply:
> Are you asking if this conditional ever evaluates to true? 

Yes, that is right. 
BTW, I found the answer to my question. The `noMissingData` initially did not seem to take into account the newly constructed share when counting the missing shares. However, it actually does where it skips the `rebuiltIndex`  while iterating over the shares of a row/col i.e., [here](https://github.com/celestiaorg/rsmt2d/blob/e8e042fdc8dcf485510c093a9c831f5b2fac2d3b/extendeddatacrossword.go#L397).